CHANGELOG.md

```diff
@@ -27,6 +27,42 @@ docker compose run --rm -it pgbackups /backup.sh
 
 (or `docker-compose` if your version of Docker does not support compose v2).
 
+## [2.1.2] 2024-02-26
+
+### Changed
+- **Breaking change**: When exporting annotations as JSON, the "features" that the annotator entered are no longer nested under `label` ([#347](https://github.com/GateNLP/gate-teamware/issues/347)).  Where previously the export would have been 
```

Comment:
Given this is technically a breaking change should we go for 2.2.0 instead of 2.1.2?